samples/dotnet/KernelHttpServer/Plugins/GitHubPlugin.cs

```diff
@@ -213,8 +214,6 @@ private async Task SummarizeCodeDirectoryAsync(string directoryPath, string sear
 
         if (filePaths != null && filePaths.Length > 0)
         {
-            this._logger.LogDebug("Found {0} files to summarize", filePaths.Length);
```

Comment:
Why remove this, it seems helpful?

Reply:
Accidental destructive debugging...I thought I'd rolled any of this back.  Thanks for the callout.